From 77cfb227d61468c46b4cbc89da37e00b6a1a93ae Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 27 Jan 2025 01:05:49 +0800
Subject: [PATCH 4/5] AOSCOS: mpv: use auto profile for hardware decoding

We don't really have any reason to use auto-safe as this is not the
default for mpv nor, say, Firefox.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 src/mpv/mpvpreview.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mpv/mpvpreview.cpp b/src/mpv/mpvpreview.cpp
index e326b630..b253e908 100644
--- a/src/mpv/mpvpreview.cpp
+++ b/src/mpv/mpvpreview.cpp
@@ -19,7 +19,7 @@ MpvPreview::MpvPreview()
     setProperty(MpvProperties::self()->Pause, true);
     setProperty(MpvProperties::self()->ReallyQuiet, true);
 
-    setProperty(MpvProperties::self()->HardwareDecoding, QStringLiteral("auto-safe"));
+    setProperty(MpvProperties::self()->HardwareDecoding, QStringLiteral("auto"));
     setProperty(MpvProperties::self()->AccurateSeek, GeneralSettings::accuratePreviewThumbnail());
     setProperty(MpvProperties::self()->AudioId, false);
     setProperty(MpvProperties::self()->AudioFileAuto, false);
-- 
2.48.1

